css/selectors/state.json

```diff
@@ -0,0 +1,49 @@
+{
+  "css": {
+    "selectors": {
+      "state": {
+        "__compat": {
+          "description": "Custom state pseudo-class selector (<code>:state()</code>)",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:state",
+          "spec_url": "https://html.spec.whatwg.org/multipage/semantics-other.html#selector-custom",
+          "support": {
+            "chrome": {
+              "version_added": false,
+              "notes": "From version 90 Chromium supports custom state selection using a dashed-ident (such as <code>:--foo</code>) instead of <code>:state()</code>."
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "121",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.element.customstateset.enabled",
+                  "value_to_set": "true"
+                }
+              ]
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
```

Comment:
Safari supports this behind the CustomStateSet flag in latest Tech Preview too

Reply:
Thanks, added!